grafana/README.md

```diff
@@ -2,3 +2,13 @@ Import these dashboards to Grafana.
 
 The steps are documented at https://grafana.com/docs/grafana/latest/dashboards/export-import/
 
+
+For the broker/trigger, run this command on the busybox to generate many requests.
+
+
+```
+for i in `seq 1 200000`; do  curl -s "http://broker-ingress.knative-eventing.svc.cluster.local/event-example/default"   -X POST   -H "Ce-Id: say-hello-goodbye"   -H "Ce-Specversion: 1.0"   -H "Ce-Type: greeting"   -H "Ce-Source: sendoff"   -H "Content-Type: application/json"   -d '{"msg":"Hello Knative! Goodbye Knative!"}'; done
+
+for i in `seq 1 200000`; do  curl -v "http://broker-ingress.knative-eventing.svc.cluster.local/event-example/default"   -X POST   -H "Ce-Id: say-goodbye"   -H "Ce-Specversion: 1.0"   -H "Ce-Type: not-greeting"   -H "Ce-Source: sendoff"   -H "Content-Type: application/json"   -d '{"msg":"Goodbye Knative!"}'; done
```

Comment:
This is giving me 500s - which I do see on the ingress dashboard which is cool.  But obviously the filter metrics and source metrics don't show anything.

Should I be seeing PingSource metrics in the dashboards if these `curl` commands aren't working?

Reply:
I didn't see behaviour like that. let me try this again

Reply:
I think we should move these `curl` commands to the `test/README.md`